namerd/examples/zk.yaml

```diff
@@ -3,6 +3,13 @@ storage:
   pathPrefix: /dtabs
   hosts:
   - localhost:2181
+  authInfo:
+    scheme: digest
+    auth: user_123:password_123
```

Comment:
per offline discussion: digest auth is fundamentally insecure because it involves sending the password in cleartext: https://zookeeper.apache.org/doc/r3.1.2/zookeeperProgrammers.html#sc_BuiltinACLSchemes

But digest is probably the easiest to show off in an example.